runtime/help/plugins.md

```diff
@@ -51,6 +50,9 @@ which micro defines:
 
 * `postinit()`: initialization function called after `init()`.
```

Comment:
While we're at it, shouldn't we improve this description: clarify that it is called after the `init()` callbacks of all other plugins have been called? Otherwise it's not clear what's the point of this `postinit()` callback.

Reply:
Absolutely, makes sense.